src/foam/box/HTTPBox.js

```diff
@@ -26,8 +26,9 @@ foam.CLASS({
   ],
 
   imports: [
+    'generator? as ctxGenerator',
```

Comment:
If your new JSON parser can properly serialize/deserialize all serializable FObjects properly and also performs faster than we should switch to it or we should optimize FON to be faster.

I don't like the idea of adding yet another knob, especially if that knob turns between "correct" and "fast."

If you really need a special serialization case in your application then I would replace MessagePortBox/Service with a custom class that uses your own serialization, only in your application.